giskard/project.py

```diff
@@ -69,7 +74,7 @@ def upload_model(
             ('requirementsFile', requirements)
         ]
         self.session.post('project/models/upload', data={}, files=files)
-        print(f"Successfully uploaded model to project '{self.project_key}'")
+        print(f"Model successfully uploaded to  project key '{self.project_key}' and is available at  {self.url} ")
```

Comment:
nitpick: double space between `at` and url. I removed a few of them last time

Reply:
same in a few other error messages, is there a particular reason to have more than 1 space?

Reply:
Nice catch ! No reason . This is extra space . I ll update it .